### PR TITLE
Refactor bootstrap to use UploadSourceStrings

### PR DESCRIPTION
**Pull request by gbalakrishnan-sonder**
This also adds a bugfix to symbolize all keys in the perform function of `UploadSourceStringsWorker` since sidekiq doesn't pass a hash with symbolized keys.

### CODE REVIEW

File: lib/tasks/upload_source_strings_for_translation.rake [hunk 1]
```diff
@@ -15,43 +15,32 @@ namespace :i18n_sonder do
...
+  task :upload_source_strings_for_object, [:model_name, :model_id] => :environment do |_task, args|
+
+    if !args[:model_name].present? || !args[:model_id].present?
+      I18nSonder.logger.error("[upload_source_strings_for_translation] Beginning upload for #{klass}")
+    else
+      klass = args[:model_name].constantize
+      id = klass.find(args[:model_id])
+
+      I18nSonder.logger.info("[upload_source_strings_for_translation] Uploaded source strings for #{total} #{klass} objects")
+      I18nSonder::Upload.new(obj).upload(I18n::default_locale)
```
Comment (apfong) [inline lib/tasks/upload_source_strings_for_translation.rake:43]: where does obj come from here?

Comment (gbalakrishnan-sonder) [reply lib/tasks/upload_source_strings_for_translation.rake:43]: clearly i haven't tested this task out yet. let me fix this.

File: lib/tasks/upload_source_strings_for_translation.rake [hunk 2]
```diff
@@ -15,42 +15,36 @@ namespace :i18n_sonder do
...
+        I18nSonder.logger.info("[upload_source_strings_for_translation] Uploaded source strings for #{total} #{klass} objects")
+      end
+    end
+  end
+
+  desc "Upload the source strings for a given model name and id"
+  task :upload_source_strings_for_object, [:model_name, :model_id] => :environment do |_task, args|
+
+    if !args[:model_name].present? || !args[:model_id].present?
+      I18nSonder.logger.error("[upload_source_strings_for_translation] Beginning upload for #{klass}")
```
Comment (apfong) [inline lib/tasks/upload_source_strings_for_translation.rake:37]: nit: should this log to something other than error? the message seems like it's a positive flow

**Review by apfong — APPROVED** [review]
lgtm

File: lib/tasks/upload_source_strings_for_translation.rake (hunk 2 above)
Comment (gbalakrishnan-sonder) [reply lib/tasks/upload_source_strings_for_translation.rake:37]: Ahh, it should be an error, the message is wrong.

**Pull request closed without merging**